src/App/App.js

```diff
@@ -28,23 +28,52 @@ function App() {
       console.log(error);
     }
   }
+  
+  function handleDelete(selectedSpell){
+    function deleteSpells() {
+      console.log('spell deleted');
+      selectedSpell.isKnown = false;
+      const filteredSpells = savedSpells.filter(spell => spell.id !== selectedSpell.id);
+      console.log(filteredSpells, 'filtered')
```

Comment:
glad you went with breaking these functions up, I think it'll make the logic flow better. one suggestion id have for this is to not forget to delete your console logs before you submit the project

Reply:
I plan to remove console logs and check for semicolons on a separate cleanup branch, thanks for the reminder!